monkey/tests/unit_tests/monkey_island/cc/repositories/utils/test_mongo_encoder.py

```diff
@@ -1,11 +1,8 @@
 import pytest
 
 from monkey_island.cc.repositories import StorageError
-from monkey_island.cc.repositories.utils import (
-    DOT_REPLACEMENT,
-    mongo_dot_decoder,
-    mongo_dot_encoder,
-)
+from monkey_island.cc.repositories.utils import mongo_dot_decoder, mongo_dot_encoder
+from monkey_island.cc.repositories.utils.mongo_encoder import DOT_REPLACEMENT
```

Comment:
Shorten the import for `mongo_encoder.`

Reply:
`DOT_REPLACEMENT` is not exported, because it's not used (and probably shouldn't be used) outside of the package